locator/abstract_replication_strategy.cc

```diff
@@ -112,10 +112,10 @@ inet_address_vector_topology_change vnode_effective_replication_map::get_pending
     return endpoints;
```

Comment:
It currently doesn't support it though, because `vnode_effective_replication_map::get_natural_endpoints()` throws if token metadata has no normal tokens. So it's incorrect to create erm with such token metadata.